lms/envs/test.py

```diff
@@ -571,3 +571,6 @@
     'JWT_ISSUER': 'https://test-provider/oauth2',
     'JWT_AUDIENCE': 'test-key',
 })
+
+# Deprecated xblocks types
+DEPRECATED_ADVANCED_COMPONENT_TYPES = []
```

Comment:
I think having test only setting is not a good idea because then it is not clear if only the tests depend on it or whether the code does too. If we need it, I would prefer it in `common.py`.